### PR TITLE
Migrate Schrodinger/transaction/bank

### PR DESCRIPTION
**Pull request by mapleFU**
Signed-off-by: mapleFU <1506118561@qq.com>

### What problem does this PR solve? 

Migrate Schrodinger/transaction/bank

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

Side effects

 - None

Related changes

 - None

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
@mapleFU Could you fix CI

File: cmd/bank/main.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	// case config
+	retryLimit  = flag.Int("retry-limit", 2, "retry count")
+	accounts    = flag.Int("accounts", 1000000, "the number of accounts")
+	interval    = flag.Duration("interval", 2*time.Second, "the interval")
+	pessimistic = flag.Bool("pessimistic", false, "use pessimistic transaction")
+	concurrency = flag.Int("concurrency", 200, "concurrency worker count")
+	longTxn     = flag.Bool("long-txn", true, "enable long-term transactions")
+	tables      = flag.Int("tables", 1, "the number of the tables")
+	// TODO: I'm not clear that this field is.
+	mode = flag.String("mode", "online", "case mode, support values: online / dev, default value: online")
```
Comment (mahjonp) [inline cmd/bank/main.go]: Delete it.

File: cmd/bank/main.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	"github.com/pingcap/tipocket/cmd/util"
+	"github.com/pingcap/tipocket/pkg/cluster"
+	"github.com/pingcap/tipocket/pkg/control"
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
+	"github.com/pingcap/tipocket/pkg/test-infra/tidb"
+	"github.com/pingcap/tipocket/pkg/verify"
+	"github.com/pingcap/tipocket/tests/bank"
+)
+
+var defaultPushMetricsInterval = 15 * time.Second
```
Comment (mahjonp) [inline cmd/bank/main.go]: What's this?

Comment (mapleFU) [reply cmd/bank/main.go]: Useless field, I'll remove it.

**Pull request merged**